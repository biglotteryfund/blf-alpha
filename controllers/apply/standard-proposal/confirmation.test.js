/* eslint-env jest */
'use strict';
const confirmationBuilder = require('./confirmation');

test.each(['england', 'northern-ireland'])(
    'should return confirmation text for %p',
    function (country) {
        const result = confirmationBuilder({
            locale: 'en',
            data: { projectCountries: [country] },
        });

        expect(result.body).toMatchSnapshot();
    }
);
