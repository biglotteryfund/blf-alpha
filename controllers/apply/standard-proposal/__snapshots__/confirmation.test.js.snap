// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return confirmation text for "england" 1`] = `
<p>
  Thanks for sending us your project proposal form.
</p>
<p>
  We appreciate you taking the time to tell us more about your project and organisation.
</p>
<h3>
  What happens next?
</h3>
<p>
  Your answers have been passed on to a funding officer, who will use this information 
to continue assessing your funding proposal.
</p>
<p>
  We’ll let you know if your proposal is something we’re interested in progressing in around 12 weeks.
</p>
<p>
  Any questions in the meantime?
  <a href="/contact">
    Contact us
  </a>
</p>
`;

exports[`should return confirmation text for "northern-ireland" 1`] = `
<p>
  We've emailed you a copy of what you wrote along
with information about what happens next.
Hold on to it in case you want to look at your answers again.
It can be helpful to refer back to if we call.
</p>
<h2>
  What happens next?
</h2>
<p>
  Your proposal has been passed on to one of our funding officers.
</p>
<p>
  We’re now prioritising proposals for COVID-19 related projects,
so they can start sooner. And it might take us longer to get
back to you about proposals that aren’t COVID-19 related.
</p>
<p>
  If it's something we can fund, you'll be asked if
you want to make a full application.
</p>
<p>
  Any questions in the meantime?
  <a href="/contact">
    Contact us
  </a>
</p>
`;
